README.md

```diff
@@ -27,8 +27,8 @@ If you don't feel like typing citations out (and let's be honest, you don't), ex
 The formats which are supported by this plugin depend on the [CAYW (Cite as you Write)](https://retorque.re/zotero-better-bibtex/citing/cayw/index.html) implementation by zotero-better-bibtex.
 
 An excerpt of the formats that are supported are as follows:
-- `cite` for latex, uses the cite command
-- `biblatex` for biblatex, uses the autocite command
+- `latex` generates natbib citations using the cite command by default (configurable, documentation above)
```

Comment:
"latex" is actually an alias for "natbib" 😄

Reply:
Ah okay, I got confused because in the CAYW documentation `natbib` is not listed as a valid format.
> The following formats are available:
>
> - `latex` Generates [natbib](https://ctan.org/pkg/natbib) citation commands. Extra URL parameters allowed:
>   - `command`: the citation command to use (if unspecified, defaults to cite)
> - `cite` is an alias for latex with the assumption you want the cite command to be cite

Reply:
Oops 😊 . Site is rebuilding to fix that.